onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -404,6 +415,7 @@ def _where_input_wrangler(
     "nn.functional.conv1d": core_ops.aten_conv1d,
     "nn.functional.conv2d": core_ops.aten_conv2d,
     "nn.functional.conv3d": core_ops.aten_conv3d,
+    "nn.functional.nll_loss": (nn_ops.aten_nll_loss, _nll_loss_input_wrangler),
```

Comment:
Please move this down so it is after nn.functional.linear for alphabetical order.

Reply:
sure.